packages/elements/src/time-picker/__test__/time-picker.test.js

```diff
@@ -325,6 +439,29 @@ describe('time-picker/TimePicker', function () {
       expect(el.value).to.equal('10:20', 'should be 10:20');
     });
 
+    it('Should able to toggle mode between 12hr and 24hr by API toggle method', async function () {
+      const el = await fixture(timePickerAMPM);
+      await elementUpdated(el);
+      let togglePart = el.renderRoot.querySelector('[part=toggle]');
+      expect(togglePart).to.exist;
+
+      expect(el.hours).to.equal(13);
+      expect(el.formattedHours).to.equal('01', 'should be 01');
+      expect(el.value).to.equal('13:30', 'should be 13:30');
```

Comment:
Other test cases cover it already. Plus, it's not the objective of this unit test.

Reply:
Updated